lib/seattleflu/id3c/api/routes.py

```diff
@@ -14,12 +14,20 @@
 blueprints.append(api_v3)
 
 @api_v1.route("/shipping/return-results/<barcode>", methods = ['GET'])
+def get_barcode_results_v1(barcode=None):
```

Comment:
Not sure we want to mark v1 as gone, we will probably use it for Cepheid results in the future?

Reply:
I assumed we would integrate Cepheid results into the v2 view.

Reply:
I was imagining the normal Flu/RSV results would be on a completely separate page than HCoV-19 results 🤷‍♀

Reply:
Oh interesting. I was thinking they'd be on the same page and we would have a `union` query joining Cepheid results and HCoV-19 results.

Reply:
Yeah, I think it's just unclear how either set of results will ultimately be returned, and this iss reflected in being unclear on what the final implementation might look like. Doing this view + API work now means we more quickly return them when decisions are made, though. I think marking the v1 endpoint 410 Gone is fine for now. We can always revert it.

Reply:
Ok, I'm going to mark v1 as gone for now since we aren't supposed to be currently using it anyway. I'm happy to revert this down the road as soon as we have more clarity.